statusHandler/termuiStatusHandler_test.go

```diff
@@ -25,25 +25,14 @@ func TestTermuiStatusHandler_TermuiShouldPass(t *testing.T) {
 	assert.NotNil(t, termuiConsole)
 }
 
-func TestTermuiStatusHandler_TestIfMetricsAreInitialized(t *testing.T) {
-	t.Parallel()
-
-	termuiStatusHandler := statusHandler.NewTermuiStatusHandler()
-
-	// check if nonce metric for example was initialized
-	_, err := termuiStatusHandler.GetTermuiMetricByKey(core.MetricNonce)
-
-	assert.Nil(t, err)
-	assert.Equal(t, 26, termuiStatusHandler.GetMetricsCount())
-}
-
 func TestTermuiStatusHandler_TestIncrement(t *testing.T) {
 	t.Parallel()
 
 	var metricKey = core.MetricNonce
 
 	termuiStatusHandler := statusHandler.NewTermuiStatusHandler()
 
+	termuiStatusHandler.SetUInt64Value(metricKey, 0)
```

Comment:
For readability, delete `var metricKey` and use `core.MetricNonce` directly as the arguments to `SetUInt64Value` here.

Reply:
done